Week3/homework/1-step3.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 function foo(func) {
-  // What to do here? 
+  console.log(func);
 }
 
 function bar() {
```

Comment:
This won't work, unfortunately.  
You are supposed to call the function and return it.
`bar` is the one that calls `console.log`.
The line should look like this:
```
return func();
```